uwblueprint/social-diversity-for-children
# Setup Database Schema and Workflow

### Notion ticket link



[Set up Prisma Workflow](https://www.notion.so/uwblueprintexecs/Set-up-Prisma-Workflow-dfa2581010db4166bfe813721b6491df)



### Implementation description
- Set up a `schema.sql` file to contain the script for creating the database tables and related indexes
- Updated README with instructions on deploying database schema



### Steps to test

1. Try running the commands in the `README.md` file to see if the database can deploy successfully



### What should reviewers focus on?

- The SQL script and the Prisma file generated from it. I went backwards and used the migrations that Prisma generated from the initial setup so we can avoid mapping. 

### Checklist
